nodejs/overview/msteams-client.md

```diff
@@ -1,61 +1,110 @@
 ---
-title: Microsoft Teams JavaScript client library
-description: Reference documentation for the latest Microsoft Teams JavaScript client library
-keywords: msteams teams client sdk javascript library reference latest
+title: Microsoft Teams SDK reference
+description: Reference documentation for Microsoft Teams SDK
+ms.date: 01/03/2023
+ms.topic: reference
+keywords: msteams live-share teamsfx teams client sdk javascript library reference latest
 ---
-# Microsoft Teams JavaScript client library
+# Microsoft Teams SDK reference
 
-Starting with *@microsoft/teams-js* version `2.0.0`, the Microsoft Teams JavaScript client library has been refactored to enable [Teams apps to run in Outlook and Office](/microsoftteams/platform/m365-apps/overview), in addition to Microsoft Teams. From a functional perspective, the TeamsJS SDK is a superset of TeamsJS 1.x versions: it supports existing Teams app functionality while adding the ability to host Teams apps in Outlook and Office.
+This article provides you with general information about the libraries that Microsoft offers to support you in the development of using The Microsoft Teams SDK.
```

Comment:
How about:

There are a number of JavaScript/TypeScript libraries available to support various aspects of Teams app development. This article provides an overview of each, along with basic installation instructions.